physical/raft/raft.go

```diff
@@ -284,6 +285,23 @@ func (b *RaftBackend) RemoveSelf() error {
 	return b.stableStore.SetUint64(removedKey, 1)
 }
 
+func (b *RaftBackend) SetPerfStandbyCleanupFunc(f func(context.Context, string) (bool, error)) {
```

Comment:
can we also rename this function and the one below it to remove references to "perf standby"s

Reply:
Sorry, I should've been more thorough with this 😞